assets/app/scripts/directives/logViewer.js

```diff
@@ -224,6 +226,25 @@ angular.module('openshiftConsole')
               goChromeless: logLinks.chromelessLink,
               restartLogs: streamLogs
             });
+
+            APIDiscovery
+              .getLoggingURL()
+              .then(function(url) {
+                var hasComponents = $scope.context.project.metadata.name && $scope.name && url;
```

Comment:
I'm thinking so as well, don't want to dig that deep into an object w/o a check.  I'll probably use `_.get($scope.context, 'project', 'metadata', 'name')`.